dwl.c

```diff
@@ -1742,9 +1744,23 @@ rendermon(struct wl_listener *listener, void *data)
 	struct timespec now;
 	clock_gettime(CLOCK_MONOTONIC, &now);
 
-	/* Do not render if any XDG clients have an outstanding resize. */
+
+			/* Do not render if any XDG clients have an outstanding resize. */
 	wl_list_for_each(c, &stack, slink) {
-		if (c->resize) {
+		if (m->resize) {
```

Comment:
Check this before the for loop

Reply:
For clarification, m->resize is only ever true, if a map/unmap has been called. Normally, I'd leave this entire part out, but if it's missing, then opening/closing something sometimes causes a flicker.

So, if we have a map/unmap event, then we use the old method using c->resize, and if not we only check if the client has properly resized yet.